### ISSUE TITLE
u_milesone x's out all the unfinished todo items when it "finishes" a milestone

### ISSUE THREAD

**Issue by sbillinge** [regro/regolith#914]
u_milestone can be used to finish milestones.  It sets the status to finished and it adds an end_date.

These days we tend to put lists of todos in the Notes field and they look like:
```
Notes:
  - () do this
  - () do that
```
These can be checked off manually by the owner as they are completed, but more or less by definition, when the milestone is completed, all the tasks must have been done, so let's have the "finisher" to turn all the `()` into `(x)` automatically so it looks like
```
Notes:
  - (x) do this
  - (x) do that
 ```

**Comment by mkwang2024** [comment]
kwang working on it

**Comment by sbillinge** [comment]
closed by #915
